src/rtcModule/webrtc.cpp

```diff
@@ -2192,11 +2328,18 @@ Session::Session(Call& call, RtMessage& packet, SdpKey sdpkey)
     mName = "sess[" + mSid.toString() + "]";
     auto wptr = weakHandle();
     mSetupTimer = setTimeout([wptr, this] {
-        if (wptr.deleted())
+        if (wptr.deleted() || mState >= kStateInProgress)
```

Comment:
To avoid problems, this timer should be cancelled if the session is setup correctly, doesn't it?
Actually, that's done for the timer that monitors the call-setup. Why not for the session-setup?